docs/plugins/play.rst

```diff
@@ -107,6 +107,15 @@ string, use ``$args`` to indicate where to insert them. For example:
 indicates that you need to insert extra arguments before specifying the
 playlist.
 
+The above example, however, does not work with current ``mpv`` because the
+``--playlist`` argument wants a different syntax. To satisfy this, the optional
+``$playlist`` can be used to meet that:
```

Comment:
```suggestion
Some players (for example ``mpv``) require a different syntax for ``--playlist``.
In that case, use the optional ``$playlist`` variable:
```

Let's try not to focus on mpv _only_ in the docs, but it is definitely worth mentioning that especially with this player it is required.